md/api.py

```diff
@@ -22,14 +22,7 @@
 
 # PURPOSE_CHOICES to be added after ODPM-31
 
-GROUP_DEFAULTS = {'ASIAN': 0,
-                  'BLACK': 0,
-                  'NATIVE AMERICAN': 0,
-                  'UNKNOWN': 0,
-                  'WHITE': 0,
-                  'HISPANIC': 0,
-                  }
-
+GROUP_DEFAULTS = {k: 0 for k in GROUPS.values()}
```

Comment:
Unless there's a reason to keep the caps version, we could get this directly from models to avoid duplication. Something like this:

```
from md.models import ETHNICITY_CHOICES
GROUP_DEFAULTS = {k: 0 for k in GROUPS.values()}
```

Though this would likely require updating the frontend code.